src/message.rs

```diff
@@ -1,77 +1,63 @@
-use std::str::FromStr;
-
-use APRSError;
-use Callsign;
-use APRSPosition;
-
-#[derive(PartialEq, Debug, Clone)]
-pub struct APRSMessage {
-    pub from: Callsign,
-    pub to: Callsign,
-    pub via: Vec<Callsign>,
-    pub data: APRSData,
+use AprsError;
+use FromStr;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct AprsMessage {
+    pub addressee: String,
+    pub text: String,
+    pub id: Option<u32>,
```

Comment:
I'll need to update this as `id` can be alphanumeric

Reply:
I'm going to open another PR to change the id to a String and add `messaging_supported` to the README. Does that work for you @Turbo87 ?